spec/index.html

```diff
@@ -837,103 +854,7 @@ <h4 id="delete"><a class="toc-anchor" href="#delete" >§</a> Delete</h4>
-<ul>
-<li>The object <strong><strong>MUST</strong></strong> contain a <code>method</code> property, and its value <strong><strong>MUST</strong></strong> be the string <code>ThreadsClose</code>.</li>
-<li>The object <strong><strong>MUST</strong></strong> contain a <code>threadId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string of the initiating Thread message.</li>
-</ul>
-</li>
-</ul>
-<h4 id="delete-2"><a class="toc-anchor" href="#delete-2" >§</a> Delete</h4>
-<pre class="language-json"><code class="language-json"><span class="token punctuation">{</span> <span class="token comment">// Message</span>
-  <span class="token property">"descriptor"</span><span class="token operator">:</span> <span class="token punctuation">{</span> <span class="token comment">// Message Descriptor</span>
-    <span class="token property">"nonce"</span><span class="token operator">:</span> <span class="token string">"9b9c7f1fcabfc471ee2682890b58a427ba2c8db59ddf3c2d5ad16ccc84bb3106"</span><span class="token punctuation">,</span>
-    <span class="token property">"method"</span><span class="token operator">:</span> <span class="token string">"ThreadsDelete"</span><span class="token punctuation">,</span>
-    <span class="token property">"threadId"</span><span class="token operator">:</span> <span class="token string">"b6464162-84af-4aab-aff5-f1f8438dfc1e"</span><span class="token punctuation">,</span>
-  <span class="token punctuation">}</span>
-<span class="token punctuation">}</span>
-</code></pre>
-<p><code>ThreadsDelete</code> messages are JSON objects that include general <a href="#message-descriptors" >Message Descriptor</a> properties and the following additional properties, which <strong><strong>must</strong></strong> be composed as follows:</p>
-<ul>
-<li>The message object <strong><strong>MUST</strong></strong> contain a <code>descriptor</code> property, and its value <strong><strong>MUST</strong></strong> be a JSON object composed as follows:
-<ul>
-<li>The object <strong><strong>MUST</strong></strong> contain a <code>method</code> property, and its value <strong><strong>MUST</strong></strong> be the string <code>ThreadsDelete</code>.</li>
-<li>The object <strong><strong>MUST</strong></strong> contain a <code>threadId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string of the initiating Thread message.</li>
+<li>The object <strong><strong>MUST</strong></strong> contain a <code>recordId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string of the record to be deleted.</li>
```

Comment:
Again `a RFC4122` vs `an RFC4122`... I still think the latter is correct.
```suggestion
<li>The object <strong><strong>MUST</strong></strong> contain a <code>recordId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string of the record to be deleted.</li>
```